docs/experimental/framebuffer_example.py

```diff
@@ -0,0 +1,246 @@
+
+    width, height, n = texture.shape
+
+    # Setting the image data
+    info = vtk.vtkInformation()
+    imageData = vtk.vtkImageData()
+    imageData.SetDimensions(width, height, 1)
+    imageData.SetNumberOfScalarComponents(n, info)
+    imageData.AllocateScalars(vtk.VTK_FLOAT, 4)
+
+    for y in range(width):
+        for x in range(height):
+            pixel = texture[y, x]
+            for i in range(n):
+                imageData.SetScalarComponentFromFloat(x, y, 0, i, pixel[i])
+
+
+
+    # Setting the texture
+    texture_obj = lib.Texture()
+    texture_obj.SetInputDataObject(imageData)
+    texture_obj.InterpolateOn()
+    texture_obj.Update()
+
+    actor.SetTexture(texture_obj)
+    actor.GetProperty().SetTexture(location, texture_obj)
+
+
+
+
+# Don't mind these shaders, I just set them up to focus first on the framebuffer, then after on the shaders themselves
+
+billboard_vert_decl =  """/* Billboard  vertex shader declaration */
+                        in vec3 center;
+                        in vec2 in_tex;
+                        
+                        out vec3 centerVertexMCVSOutput;
+                        out vec3 normalizedVertexMCVSOutput;
+                        varying vec2 out_tex;"""
+
+billboard_vert_impl =  """/* Billboard  vertex shader implementation */
+                        centerVertexMCVSOutput = center;
+                        normalizedVertexMCVSOutput = vertexMC.xyz - center; // 1st Norm. [-scale, scale]
+                        float scalingFactor = 1. / abs(normalizedVertexMCVSOutput.x);
+                        float size = abs(normalizedVertexMCVSOutput.x) * 2;
+                        normalizedVertexMCVSOutput *= scalingFactor; // 2nd Norm. [-1, 1]
+                        vec2 billboardSize = vec2(size, size); // Fixes the scaling issue
+                        vec3 cameraRightMC = vec3(MCVCMatrix[0][0], MCVCMatrix[1][0], MCVCMatrix[2][0]);
+                        vec3 cameraUpMC = vec3(MCVCMatrix[0][1], MCVCMatrix[1][1], MCVCMatrix[2][1]);
+                        vec3 vertexPositionMC = center +
+                            cameraRightMC * billboardSize.x * normalizedVertexMCVSOutput.x +
+                            cameraUpMC * billboardSize.y * normalizedVertexMCVSOutput.y;
+                        out_tex = in_tex;
+                        gl_Position = MCDCMatrix * vec4(vertexPositionMC, 1.);"""
+
+billboard_frag_decl ="""                   
+                    varying vec2 out_tex;
+                    
+                    uniform sampler2D screenTexture;"""
+                    
+                    
+billboard_frag_impl ="""                   
+                        vec3 texture = texture(screenTexture, out_tex);
+                        vec3 color = vec3(1.0, 0.0, 0.0);
+                        
+                        fragOutput0 = vec4(color, 1.0);"""
+
+
+frag_decl = """varying vec2 out_tex; 
+               uniform vec2 res0;  
```

Comment:
@JoaoDell  you should declare `in vec3 normalizedVertexMCVSOutput;` here

```python
frag_decl = """varying vec2 out_tex;
               in vec3 normalizedVertexMCVSOutput;
               uniform vec2 res0;
               uniform vec3 point0;"""
```

Reply:
It seems like the problem was somewhere else, but i have just updated the PR with some changes that don't conflict with the shaders